minidump-stackwalk/stackwalker.cc

```diff
@@ -976,7 +995,7 @@ static void ConvertMemoryInfoToJSON(Minidump& dump,
     }
   }
 
-  root["largest_free_vm_block"] = ToHex(largest_free_block);
+  root["largest_free_vm_block"] = ToHexAddress(largest_free_block, arch_bits);
```

Comment:
I don't know what `largest_free_vm_block` is, but I looked at a bunch of values and it looks like a memory address and not an offset so I adjusted it. Was that right? Is it right but not helpful and we shouldn't pad it?

Reply:
It is a size in bytes so there's no need to pad it; but it would be better to print it in decimal format instead.